go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/aws/amazon-ecs-cni-plugins
+
+go 1.22.0
+
+require (
+	github.com/aws/aws-sdk-go v1.5.11-0.20161122232317-92ed7a76d078
+	github.com/cihub/seelog v0.0.0-20161009225354-175e6e3d439f
+	github.com/containernetworking/cni v0.0.0-00010101000000-000000000000
+	github.com/docker/libkv v0.2.2-0.20211217151845-dfacc563de57
+	github.com/golang/mock v1.6.0
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.9.0
+	github.com/vishvananda/netlink v0.0.0-20170524205439-99091d844046
+	golang.org/x/tools v0.24.0
```

Comment:
`go mod init` is supposed to populate go.mod from [Godeps.json](https://github.com/aws/amazon-ecs-cni-plugins/blob/master/Godeps/Godeps.json), but it wasn't able to for this project. I believe it failed because it couldn't find `0db5cd5` in [containernetworking/cni](https://github.com/containernetworking/cni). That commit is actually from a fork -- https://github.com/aaithal/cni/commit/0db5cd54f92d60b16f429cc9458cadf1cb56adc0.

To add the required dependencies at those versions, I ran
```
go get github.com/aws/aws-sdk-go@92ed7a76d078
go get github.com/cihub/seelog@175e6e3d439f
...
```
The commit SHAs I used as the versions are from [Godeps.json](https://github.com/aws/amazon-ecs-cni-plugins/blob/master/Godeps/Godeps.json). (Semver release tags are preferred but I used SHAs here so as to not change too many dependencies with the migration.)

The versions of indirect dependencies were automatically set by Go modules (looks like it defaults to the latest release versions), but if we want them to be the same as what's in [Godeps.json](https://github.com/aws/amazon-ecs-cni-plugins/blob/master/Godeps/Godeps.json), we can downgrade them with
```
go get github.com/coreos/go-iptables@{VERSION}
go get github.com/davecgh/go-spew@{VERSION}
...
```